tox.ini

```diff
@@ -0,0 +1,15 @@
+[tox]
+envlist = py27-django111
```

Comment:
@moeez96 We need to configure tox for both py27 and py35. Please configure tox for py35 as well.

Reply:
also add allow_failures in travis config for py35 to ignore py35 failures for now.

Reply:
@ihtram py35 is not a part of travis env hence adding allow_failures in travis for a tox config doesnt seems well. How about we keep py27 for this pr in tox and add py35 in tox with the next PR which actually contains all py35 tests. What do you think?

Reply:
we should configure py35 for travis. Adding tox for only py27 is meaningless as the purpose of tox is to run multiple environments. After configuring tox for both versions we can then start fixing broken tests.